tests/templates/kuttl/logging/01-install-hbase-vector-aggregator.yaml

```diff
@@ -5,7 +5,7 @@ commands:
   - script: >-
       helm install hive-vector-aggregator vector
       --namespace $NAMESPACE
-      --version 0.39.0
+      --version 0.42.1
```

Comment:
```suggestion
      --version 0.46.1
```

Reply:
This is correct. I updated the PR description to mention both versions.